src/data/staking-products.json

```
@@ -403,6 +403,49 @@
         "eventName": "Clicked stakefish go to link"
       }
     },
+    {
+      "name": "P2P.org",
+      "imageName": "P2P",
+      "hue": 227,
+      "launchDate": "2023-01-10",
+      "url": "https://p2p.org/networks/ethereum/",
+      "audits": [
+        {
+          "name": "Mixbytes (v1)",
+          "url": "https://github.com/mixbytes/audits_public/blob/79cd8153bfb6098227b43dfba8d061881a94a248/P2P.org/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20(v.1)/P2P.org_ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20Smart%20Contracts%20Audit%20Report.pdf"
+        },
+        {
+          "name": "Mixbytes (v2)",
+          "url": "https://github.com/mixbytes/audits_public/blob/79cd8153bfb6098227b43dfba8d061881a94a248/P2P.org/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20(v.2)/ETH2%20Depositor%20%26%20ETH%20Staking%20Fee%20Distributor%20Security%20Audit%20Report%20(v.2).pdf"
+        }
+      ],
+      "minEth": 32,
+      "additionalStake": null,
+      "additionalStakeUnit": null,
+      "monthlyFee": 5,
+      "monthlyFeeUnit": "%",
+      "isFoss": false,
+      "hasBugBounty": false,
+      "isTrustless": true,
+      "isPermissionless": true,
+      "pctMajorityExecutionClient": 50,
+      "pctMajorityConsensusClient": 30,
+      "isSelfCustody": true,
+      "platforms": ["Browser", "Linux", "macOS", "Windows", "iOS", "Android"],
```

If everything is through the browser, I would just use Browser here... the others are meant for native, non-web applications

```suggestion
      "platforms": ["Browser"],
```